dali/image/jpeg_mem.cc

```diff
@@ -593,10 +593,9 @@ bool GetImageInfo(const void* srcdata, int datasize, int* width, int* height,
   SetSrc(&cinfo, srcdata, datasize, false);
 
   DALI_ENFORCE(jpeg_read_header(&cinfo, TRUE) == JPEG_HEADER_OK);
-  DALI_ENFORCE(jpeg_start_decompress(&cinfo));  // required to transfer image size to cinfo
```

Comment:
the original code says that this is required to transfer the image size to cinfo. Are we sure it is not needed?

Reply:
It was required to populate output_width, output_height and output_components, but is not needed for image_width, image_height and num_components. (Operator tests still passes).